File: javascript/2388.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var n = parseInt(lines.shift()), total = 0

for(let i = 0; i < n; i++) {
    var [t, v] = lines.shift().split(' ').map(e => parseInt(e))
    total += t * v
}

console.log(total)

File: C++/2388.cpp
#include <iostream>

using namespace std;

int main() {
    
    int n, i, t, v, total = 0;
    cin >> n;
    
    for(i = 0; i < n; i++) {
        cin >> t >> v;
        
        total += t * v;
    }
    cout << total << endl;
    
    return 0;
}